src/initialize_wizard.cpp

```diff
@@ -274,6 +274,7 @@ InitializeWizard::InitializeWizard()
 {
     // Window properties.
     setWindowTitle("Setup Obliteration.");
+    setWizardStyle(QWizard::ModernStyle);
```

Comment:
We should explicitly set this on Windows only so on the other platform it will still looks native.